jwst/lib/set_telescope_pointing.py

```diff
@@ -413,37 +425,33 @@ def update_wcs_from_telem(
 
     # Update Aperture pointing
     model.meta.aperture.position_angle = wcsinfo.pa
-    model.meta.wcsinfo.crval1 = wcsinfo.ra
-    model.meta.wcsinfo.crval2 = wcsinfo.dec
     model.meta.wcsinfo.ra_ref = wcsinfo.ra
     model.meta.wcsinfo.dec_ref = wcsinfo.dec
     model.meta.wcsinfo.roll_ref = compute_local_roll(
-        vinfo.pa, wcsinfo.ra, wcsinfo.dec, siaf.v2ref, siaf.v3ref
+        vinfo.pa, wcsinfo.ra, wcsinfo.dec, siaf.v2_ref, siaf.v3_ref
     )
-    (model.meta.wcsinfo.pc1_1,
-     model.meta.wcsinfo.pc1_2,
-     model.meta.wcsinfo.pc2_1,
-     model.meta.wcsinfo.pc2_2) = calc_rotation_matrix(
-         wcsinfo.pa * D2R, vparity=siaf.vparity
-     )
-
-    wcsaxes = model.meta.wcsinfo.wcsaxes
-    if wcsaxes is None:
-        wcsaxes = 2
-    model.meta.wcsinfo.wcsaxes = max(2, wcsaxes)
+    if model.meta.exposure.type.lower() in TYPES_TO_UPDATE:
```

Comment:
TYPES_TO_UPDATE includes the FGS guide star modes. Is there a separate branch somewhere in the middle here (that isn't visible in the diff) that does the special WCS computation for those data sets? They are handled differently than normal science exposures.

Reply:
I removed the guider modes from TYPES_TO_UPDATE